README.md

```diff
@@ -2,7 +2,7 @@
 
 OSS Leaderboard shows the top contributors based on their contribution activities on GitHub.
 
-<img src = "https://i.imgur.com/SPMKoJt.png" width = "700">
+<img src = "https://imgur.com/lxiff4i" width = "700">
```

Comment:
One more thing - let's update the image to reflect the latest design.

Reply:
I have updates the link as well will validate it as well.